analytical/templatetags/piwik.py

```diff
@@ -43,7 +43,31 @@
 
 VARIABLE_CODE = '_paq.push(["setCustomVariable", %(index)s, "%(name)s", "%(value)s", "%(scope)s"]);'  # noqa
 IDENTITY_CODE = '_paq.push(["setUserId", "%(userid)s"]);'
-DISABLE_COOKIES_CODE = '_paq.push([\'disableCookies\']);'
+DISABLE_COOKIES_CODE = "_paq.push(['disableCookies']);"
+
+GIVE_CONSENT_CLASS = "piwik_give_consent"
+REMOVE_CONSENT_CLASS = "piwik_remove_consent"
+ASK_FOR_CONSENT_CODE = """
+_paq.push(['requireConsent']);
+
+var elements = document.getElementsByClassName("{}");
+for (var i = 0; i < elements.length; i++) {{
+    elements[i].addEventListener("click",
+      function () {{
+        _paq.push(["forgetConsentGiven"]);
+      }}
+    );
+}}
+
+var elements = document.getElementsByClassName("{}");
+for (var i = 0; i < elements.length; i++) {{
+    elements[i].addEventListener("click",
+      function () {{
+        _paq.push(["rememberConsentGiven"]);
+      }}
+    );
+}}
+""".format(REMOVE_CONSENT_CLASS, GIVE_CONSENT_CLASS)
```

Comment:
We should really just keep the Piwik implementation as is. Not extend it.

I'm not even sure if it would work. There is no Piwik that does consent. It's only Matomo that does it.